Tasks/zip.js

```diff
@@ -1,18 +1,26 @@
 // Zip two arrays, [1, 2] and [3, 4] -> [[1, 3], [2, 4]]
+'use strict';
 
-const zip = function (a = [], b = []) {
-  let i = 0;
-  j = 0;
-  for (x of b) {
-    CELL = [a[i++], x];
-    if (i < j) {
-      delete a[i++];
-    } else {
-      (() => (b[j++] = CELL))();
-    }
-    if (CELL[0] == undefined) b.length -= 1
+const zip = (arrayA = [], arrayB = []) => {
+
+  const firstArray = arrayA.slice(0);
+  const arrayResult = arrayB.slice(0);
```

Comment:
We do not need to clone incoming arrays just create outgouing

Reply:
Here are fixed hardcoded keys, removed clone of array and added indentation.

https://github.com/VladiusVostokus/Exams_SEF1/blob/exams-IM-24-%D0%92%D0%BB%D0%B0%D0%B4%D0%B8%D1%81%D0%BB%D0%B0%D0%B2-%D0%9A%D0%BE%D0%B2%D0%B0%D0%BB%D0%B5%D0%BD%D0%BA%D0%BE/Tasks/only.js